src/SignalR/server/Specification.Tests/src/Microsoft.AspNetCore.SignalR.Specification.Tests.csproj

```diff
@@ -16,9 +16,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <!-- This reference needs to be moved. https://github.com/aspnet/AspNetCore/issues/6581 -->
-    <Reference Include="Microsoft.AspNetCore.Testing" />
```

Comment:
Would privateassets=true fix it and remove the code dupe?

Reply:
You would get runtime errors about missing Microsoft.AspNetCore.Testing.dll.